net-misc/xmrig/xmrig-6.22.2-r1.ebuild

```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit cmake systemd
 
-DESCRIPTION="RandomX, CryptoNight, KawPow, AstroBWT, and Argon2 CPU/GPU miner"
-HOMEPAGE="https://xmrig.com https://github.com/xmrig/xmrig"
```

Comment:
As I've said before, I don't agree with going around removing this. Leave it up to maintainers.